Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -609,6 +609,8 @@ def project(pts, projection, wc, geo=None):
         pname = projDict.get(projection._type, projection.type)
         projName = pname
         pd.SetName(projName)
+        pd.SetOptionalParameter("over", "true")
+        pd.SetCentralMeridian(float(xm + xM) / 2.0)
```

Comment:
> @danlipsa what about case where user specified the central meridian?

@doutriaux1 that what was the question @danlipsa asked in the issue. How does someone specify that in vcs?